apps/vs-code-extension/project.json

```diff
@@ -16,7 +16,6 @@
         "external": ["vscode"],
         "platform": "node",
         "format": ["cjs"],
-        "minify": true,
```

Comment:
Well... There we probably have the case where we want to distinguish different environments?

Reply:
Yeah, that would be the best on the long run. I can look into it.